tests/integration/interactive_init_test.go

```diff
@@ -64,6 +64,29 @@ var _ = Describe("odo init interactive command tests", func() {
 		Expect(helper.ListFilesInDir(commonVar.Context)).To(ContainElements("devfile.yaml"))
 	})
 
+	It("should fail when an invalid component name is passed", func() {
+		command := []string{"odo", "init"}
+		_, err := helper.RunInteractive(command, nil, func(ctx helper.InteractiveContext) {
+
+			helper.ExpectString(ctx, "Select language")
+			helper.SendLine(ctx, "go")
+
+			helper.ExpectString(ctx, "Select project type")
+			helper.SendLine(ctx, "\n")
```

Comment:
Please send `""` instead of `"\n"` as the newline is already handled by SendLine (see https://github.com/redhat-developer/odo/pull/6091)